### ISSUE TITLE
[BUG] TFT: Target Included in Historic Encoder

### ISSUE THREAD

**Issue by yesteryearer** [sktime/pytorch-forecasting#1820]
I'm busy implementing a TFT for a use case where the target variable will be unavailable during inference time. At first I assumed that if the target is excluded from the `time_varying_unknown_reals` that the past encoder doesn't consider it all, but upon further reading, it seems that the target variable is handled differently.

Is there any clean way to configure the TFT or TimeSeriesDataset to explicitly make it avoid any notion of the past targets? In my use-case specifically we want to only predict one step into the future.

I also found [this thread](https://github.com/sktime/pytorch-forecasting/issues/1585) for which I can't deduce anything conclusive.

**Comment by fkiraly** [comment]
FYI @jdb78, any insights?

**Comment by jdb78** [comment]
The target variable is generally not used in inference but only for training and evaluation. If you add it separately to unknown reals, it will use historic data but make a cutoff for inference.

There is one exception and that is normalization. So make sure to use a prefitted target normalizer or the encodernormalizer. That stuff happens automatically if you use construct your test dataset from the training dataset with .from_dataset

**Comment by yesteryearer** [comment]
Hi @jdb78, thanks for the confirmation. 

On your comment, If I use a target normalisation, will this effectively then be 'baked in' to the model or would I need to supply the normalisation weights using historic data? For instance, if I'm doing inference with a model trained using GroupNormalizer, is this normalisation then associated with the specific group and baked in or will it be determined based on the new input data  — assuming all group ids are known?
